### DISCUSSION TITLE
organization name already been taken while that name was taken by me previously and deleted recently

### DISCUSSION THREAD

**Discussion by recursivezero** [community/community#83923] [New to GitHub]
I have recently delete organization and later delete it and again now want to create organization name with same name but it says already been taken. How long I have to wait for that?

**Comment by MPS-SteveGornick** [comment]
When deleting an organization, it warns saying that the name will not be available for 90 days.

**Reply by xkeshav** [reply]
Oh is it? Thanks for the info. I didn't read cautiously while deleting.

**Reply by MPS-SteveGornick** [reply]
I edited my reply.  I re-checked and see It shows 90 days (and not the lengthier time I had previously).
